src/python/library/tritonclient/http/_utils.py

```diff
@@ -35,16 +35,15 @@ def _get_error(response):
     indicates the error. If no error then return None
     """
     if response.status_code != 200:
-        body = response.read()
+        body = response.read().decode()
```

Comment:
We probably need to wrap this in try/catch block as well and do a similar handling like we do for `JSONDecodeError`.